pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -144,6 +144,15 @@ func add(mgr manager.Manager, c controller.Controller) error {
 		return fmt.Errorf("intrusiondetection-controller failed to watch the Secret resource: %v", err)
 	}
 
+	// These watches are here to catch a modification to the resources we create in reconcile so the changes would be corrected.
```

Comment:
I am just commenting in a random line. I do not see anything in this PR that routes Kibana requests through es-gateway. I was under the impression that this would be part of the work.

Reply:
The components that need to reach Kibana can do so through the new `tigera-secure-es-gateway-http` service. I changed the endpoint in `pkg/render/common/kibana/service.go` which is what components use to set their KIBANA_HOST value.